operations/enterprises.md

```diff
@@ -678,17 +679,24 @@ Adds a new task to the enterprise, optionally to a specified department.
 | `ClientToken` | string | required | Token identifying the client application. |
 | `AccessToken` | string | required | Access token of the client application. |
 | `Client` | string | required | Name and version of the client application. |
-| `DepartmentId` | string | optional | Unique identifier of the [Department](enterprises.md#department) the task is addressed to. |
 | `Name` | string | required | Name \(or title\) of the task. |
 | `Description` | string | optional | Further decription of the task. |
 | `DeadlineUtc` | string | required | Deadline of the task in UTC timezone in ISO 8601 format. |
+| `ServiceOrderId` | string | optional | Unique identifier of the Service Order (for example a [Reservation](reservations.md#reservation)) the task is linked with. |
```

Comment:
There should be possible send Id of the product service order from https://mews-systems.gitbook.io/connector-api/operations/services#add-order
I would name it just order and write all examples

Reply:
Will link the `Add order` operation itself, because direct link to response looks like this `services#response-10` which would get scrambled in future.